### ISSUE TITLE
Error in.Call

### ISSUE THREAD

**Issue by samendrasherchan** [USGS-R/dataRetrieval#381]
https://owi.usgs.gov/R/dataRetrieval.html#8

tried this but i got this error Error in .Call("parse_dt", x, format, TRUE, FALSE) :
 "parse_dt" not resolved from current namespace (lubridate)

**Comment by wdwatkins** [comment]
Which specific line causes the error?

**Comment by wdwatkins** [comment]
And I would suggest first of all updating your packages.

**Comment by samendrasherchan** [comment]
I followed the powerpoint https://owi.usgs.gov/R/dataRetrieval.html#10 
and after this, there is an error 
siteNo <- "05427718"
pCode <- "00060"
start.date <- "2014-10-01"
end.date <- "2015-09-30"

yahara <- readNWISuv(siteNumbers = siteNo,
                     parameterCd = pCode,
                     startDate = start.date,
                     endDate = end.date)

**Comment by wdwatkins** [comment]
That works for me.  I suspect you need to update your packages  (run `update.packages()` or use the button in RStudio).  Could you try that, and if it still doesn't work, give us the output of `sessionInfo()`?

**Comment by samendrasherchan** [comment]
I updated and it worked! Is there a way I can get all physical data from selected counties?

**Comment by wdwatkins** [comment]
If you know what parameter codes you want, and the FIPs codes of the counties you want, yes, e.g.
```
va_counties <- c("51001","51003","51005","51007","51009","51011","51013","51015")
va_data <- readNWISdata(
                                 countyCd=va_counties,
                                 parameterCd = c("00060","00010"),
                                 startDate = "2015-01-01",
                                 endDate = "2015-01-30", service= "iv")
```
You can get counties by name too, but I think only one at a time.  Check out the documentation for `readNWISdata` (e.g. `?readNWISdata`) for some more examples.  You may also find the `countyCdLookup` function and the built-in data `parameterCdFile` useful.

**Comment by samendrasherchan** [comment]
I tried this after searching all FIPs codes of the counties I'm interested: 
fl_counties <- c("12053","12129","12131","12017","12081","12013","12103","12037","12101","12029","12033",
                 "12087","12071","12005","12065","12075","12091","12021","12115","12123","12063","12113","12045",
                 "12057","12015")
fl_data <- readNWISdata(
  countyCd=fl_counties,
  parameterCd = c("00060","00010"),
  startDate = "2015-01-01",
  endDate = "2015-01-30", service= "iv")
fl_data
I got an error. Request failed [400]. Retrying in 1 seconds...
For: https://nwis.waterservices.usgs.gov/nwis/iv/?countyCd=12053,12129,12131,12017,12081,12013,12103,12037,12101,12029,12033,12087,12071,12005,12065,12075,12091,12021,12115,12123,12063,12113,12045,12057,12015&parameterCd=00060,00010&startDT=2015-01-01&endDT=2015-01-30&format=waterml,1.1&

Error in getWebServiceData(obs_url, encoding = "gzip") : 
  Bad Request (HTTP 400).

Is there a limit on how many counties we can run at one time?
